db/migrate/20151113173639_fix_book_author_relation.rb

```diff
@@ -0,0 +1,15 @@
+class FixBookAuthorRelation < ActiveRecord::Migration
+  def up
+    remove_column :authors, :books_id
+    remove_column :authors, :author_id
+    add_column :books, :author_id, :integer
+    add_foreign_key :books, :authors, on_delete: :cascade
+  end
+
+  def down
+    add_column :authors, :books_id, :integer
+    add_column :authors, :author_id, :integer
+    remove_column :books, :author_id
+    remove_foreign_key :books, :authors
```

Comment:
I'm not sure it's necessary here, you should remove foreign key before removing column, but if you just remove the column, then db will remove foreign key automatically that associates with that column.